speedy-recovery-client/src/components/secured/messaging/MessagingPage.js

```diff
@@ -34,21 +42,31 @@ class MessagingPage extends Component {
     return (
       <div>
         <h1>Messaging </h1>
-
         {chatItems}
       </div>
     );
   }
 
   componentDidMount() {
-    // TODO: Update state when data from back-end is available instead
-    this.convertAndSetData(exampleConversations);
-  }
-
-  convertAndSetData(conversationData) {
-    // TODO: Access actual back-end data, consider missing values for optional fields
-    const conversations = conversationData;
-    this.setState({ conversations });
+    if(this.props.user){
```

Comment:
Rather than doing all of this in componentDidMount, extract this as a helper function